pkg/runtime/core/errors_test.go

```diff
@@ -0,0 +1,35 @@
+package core_test
+
+import (
+	"testing"
+
+	"github.com/esell/ferret/pkg/runtime/core"
```

Comment:
You can clone your fork using project's original path. In this case, you won't need to deal with paths.